fleet_odometer_km_count/models/fleet_vehicle_odometer.py

```diff
@@ -0,0 +1,20 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+class FleetVehicleOdometer(models.Model):
+    _inherit="fleet.vehicle.odometer"
+
+    initial_km=fields.Float()
+    km_diference=fields.Float(compute="_compute_final_odometer_count", store=True)
+    value = fields.Float()
```

Comment:
Change name fields, and the field calculate is value:
```suggestion
    km_start = fields.Float()
    km_end = fields.Float()
    value = fields.Float(compute="_compute_value", store=True)
```

Reply:
I did like this before, but  `value` is the total of km in the vehicle. Also `fleet.vehicle` use this variable and maybe we don´t need to be changing always that.
![Selección_001](https://github.com/user-attachments/assets/3554a76a-c74b-488e-b21f-6f15f4c47b4a)
![Selección_002](https://github.com/user-attachments/assets/798c953e-ea9e-40b9-b386-2fd5b53731a3)

Reply:
Okay... But this field not is km_end should be the km_start, then create new field km_end and when u change km_end then recalculate in the new field km_difference